lib/plugins/inspec-init/templates/plugins/inspec-plugin-template/inspec-plugin-template.gemspec

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+
+# As plugins are usually packaged and distributed as a RubyGem,
+# we have to provide a .gemspec file, which controls the gembuild
+# and publish process.  This is a fairly generic gemspec.
+
+# It is traditional in a gemspec to dynamically load the current version
```

Comment:
There are many points where Ruby may need to examine a gemspec without installing it, activating it, or otherwise committing to its presence. The link you refer to discusses adding lib/ during gem activation.

Here, in order to examine the gemspec (which contains live code to read the version number), we must find the version file, so we need `lib/` to find it.